src/main/resources/templates/nopermission.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:th="http://www.thymeleaf.org">
+	<head>
+		<meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+		<title>National Cybersecurity Laboratories - Error</title>
+		<object th:include="fragments/header :: header" th:remove="tag" />
+	</head>
+    <body class="mybg loginbackground">
+        <div th:replace="fragments/dashboard_navigation :: dashboardnav">&nbsp;</div>
+        <div class="container">
+            <div class="row">
+				<p>&nbsp;</p>
+				<section id="joinUs" class="join-us">
+					<div class="container" >
+						<div class="row">
+							<div class="box effect5">
+								<div class="heading text-center col-sm-12">
+									<p>&nbsp;</p>
+									<p>&nbsp;</p>
+									<div class="col-sm-2"></div>
+									<div class="col-sm-3"><img src="images/error-img1.png" th:src="@{~/images/error-img1.png}" width="249" height="377"/></div>
+									<p>&nbsp;</p>
+									<p>&nbsp;</p>
+									<div class="col-sm-7 col-offset-1">
+										<p align="left" class="h30">OOOOPS !</p>
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this deprecated "align" attribute. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3AS1827)